client/state/billing-transactions/transaction-filters/reducer.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import {
+	BILLING_TRANSACTIONS_FILTER_SET_APP,
+	BILLING_TRANSACTIONS_FILTER_SET_DATE,
+	BILLING_TRANSACTIONS_FILTER_SET_PAGE,
+	BILLING_TRANSACTIONS_FILTER_SET_QUERY,
+} from 'state/action-types';
+import { combineReducers, createReducer, keyedReducer } from 'state/utils';
+
+/**
+ * Returns the updated app filter state after an action has been dispatched
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {String}        Updated state
+ */
+export const app = createReducer( '', {
```

Comment:
Any good reason to have `app` default to an empty string, rather than `null` or `undefined`?